test/steps/kogitoruntime.go

```diff
@@ -49,10 +50,18 @@ func registerKogitoRuntimeSteps(s *godog.Suite, data *Data) {
 }
 
 // Deploy service steps
-
 func (data *Data) deployExampleServiceFromRuntimeRegistryWithConfiguration(runtimeType, kogitoApplicationName string, table *messages.PickleStepArgument_PickleTable) error {
 	imageTag := data.ScenarioContext[getBuiltRuntimeImageTagContextKey(kogitoApplicationName)]
+	return data.deployExampleServiceFromImageWithConfiguration(runtimeType, kogitoApplicationName, imageTag, table)
+}
+
+// Can be renamed to deployExampleServiceWithConfiguration once KogitoApp is removed
+func (data *Data) deployRuntimeExampleServiceWithConfiguration(runtimeType, kogitoApplicationName string, table *messages.PickleStepArgument_PickleTable) error {
+	// Passing empty image tag so image values are not filled
+	return data.deployExampleServiceFromImageWithConfiguration(runtimeType, kogitoApplicationName, "", table)
```

Comment:
Question: if the image is empty, the operator will try to match the image stream with same name? The KogitoBuild will have created the image stream previously. 
Is this correct?

Reply:
Yes, correct.

Reply:
Ok